Edit tags, final batch.

dbrant opened the pull request:
https://phabricator.wikimedia.org/T366732

cooltey commented:
I see this error message when trying to submit an image tag.
<img src="https://github.com/user-attachments/assets/4d253521-17de-4010-8b85-ac6b5fcf465c" width="450" />

The other issue was: the "Publishing" overlaps the error message so there's no way to cancel the process so I have to press the system back button.

dbrant commented:
> I see this error message when trying to submit an image tag.

Good catch, fixed.

> The other issue was: the "Publishing" overlaps the error message so there's no way to cancel the process so I have to press the system back button.

Also fixed -- this was because we were using the same `CoroutineExceptionHandler` for two different state flows.
